html/index.html

```diff
@@ -6,7 +6,55 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
 </head>
-<body>
-    
-</body>
+  <body>
+    <header>
+      <h1>Marte Auer</h1>
+      <nav id="nav">
+        <a href="#about-me">About Me</a>
+        <a href="#contact">Contact</a>
+        <a href="#skills">Skillset</a>
```

Comment:
```suggestion
<ul>
        <li><a href="#about-me">About Me</a></li>
         <li><a href="#contact">Contact</a></li>
         <li><a href="#skills">Skillset</a></li>
 </ul> 
```
whenever you see something that kinda looks like a list-> it should be a list!